consensus_encoding/src/encode/encoders.rs

```diff
@@ -147,8 +144,15 @@ impl<A, B, C, D, E, F> Encoder6<A, B, C, D, E, F> {
     }
 }
 
-impl<'e, A: Encoder<'e>, B: Encoder<'e>, C: Encoder<'e>, D: Encoder<'e>, E: Encoder<'e>, F: Encoder<'e>> Encoder<'e>
-    for Encoder6<A, B, C, D, E, F>
+impl<
+        'e,
+        A: Encoder<'e>,
+        B: Encoder<'e>,
+        C: Encoder<'e>,
+        D: Encoder<'e>,
+        E: Encoder<'e>,
+        F: Encoder<'e>,
+    > Encoder<'e> for Encoder6<A, B, C, D, E, F>
```

Comment:
WTF. Can't win em all I suppose.

Reply:
Yeah, I was a little frustrated by this, but not enough to stick a `rustfmt::skip` or something into it.